open-hackathon-client/src/client/static/js/views/oh.site.workspace.js

```diff
@@ -133,8 +141,8 @@
                     purl: "",
                     imgUrl: data.remote_servers[i].name == 'cloud_eclipse' ? '/static/pic/idehub.png' : '/static/pic/dseries.png',
                     name: data.remote_servers[i].name,
-                    surl: data.remote_servers[i].name == 'cloud_eclipse' ? data.remote_servers[i].url + window.location.origin : data.remote_servers[i].url + "&oh=" + $.cookie('token')
-                })
+                    surl: data.remote_servers[i].name == 'cloud_eclipse' ? data.remote_servers[i].url + window.location.origin : data.remote_servers[i].guacamole_host + "/guacamole/#/client/" + get_guacamole_server_name(data.remote_servers[i].name, "c", "openhackathon") + "?oh=" + $.cookie('token') + "&name=" + data.remote_servers[i].name
+		    })
```

Comment:
still more `tab`s found.  You can do following to avoid `tab`:
- Change the settings of your editor/tools to always replace `tab` with spaces
- Show all white spaces with dot. After that, it's much easier to distinguish `tab` with `space` in naked eyes.